index.js

```diff
@@ -1,48 +1,90 @@
 'use strict'
+/**
+ * @typedef {import('unified').Settings} Options
+ * @typedef {import('unified').FrozenProcessor} FrozenProcessor
+ * @typedef {import('unified').RunCallback} RunCallback
+ * @typedef {import('unified').Transformer} Transformer
+ * @typedef {import('unist').Node} Node
+ */
 
 var hast2mdast = require('hast-util-to-mdast')
 
-module.exports = attacher
+/**
+ * Attacher.
+ *
+ * If a destination is given, runs the destination with the new mdast
+ * tree (bridge-mode). Without destination, returns the mdast tree: further
+ * plugins run on that tree (mutate-mode).
+ *
+ */
+const attacher =
```

Comment:
For this to be semver minor, we'd want to keep ES5 compatibility.

```suggestion
var attacher =
```

The next major release, will likely move to ESM, `let`, `const`, and the like.

Reply:
Oops. Force of habit. Thanks!